stan/math/prim/err/elementwise_check.hpp

```diff
@@ -203,7 +196,7 @@ inline void elementwise_check(const F& is_good, const char* function,
                               const Indexings&... indexings) {
   for (size_t j = 0; j < x.size(); j++) {
     elementwise_check(is_good, function, name, x[j], must_be, indexings..., "[",
-                      j, "]");
+                      j + 1, "]");
```

Comment:
Why the + 1 here?

Does this have anything to do with Stan indexing by 1 or not? There's a variable for this here: https://github.com/stan-dev/math/blob/235090a31ccdabdd0d9d35bfb77ba3d4b984590f/stan/math/prim/meta/error_index.hpp

I think by default the Math library compiles as if it is indexing by 1, and you can optionally change this variable.

Reply:
Because I removed the +1 from `internal::pipe_in`. Yeah I can change this to instead use + error_index::value.